README.md

```diff
@@ -824,6 +825,224 @@ and Equinox will supply the _initial_ value for the `project` function to render
 > passing that state to the `decider` function, which, assuming it's implemented in an [_idempotent_](https://en.wikipedia.org/wiki/Idempotence)
 > manner, will indicate that there are no events to be written.
 
+<a name="what-is-a-decider"/></a>
+### What is a Decider? How does the Equinox `type Decider` relate to Jérémie's concept of one? :pray: [@rmaziarka](https://github.com/rmaziarka)
+
+The single best treatment of the concept of a Decider that's online at present is [this 2h45m video](https://www.youtube.com/watch?v=kgYGMVDHQHs) on [Event Driven Information Systems](https://www.youtube.com/channel/UCSoUh4ikepF3LkMchruSSaQ) with [Jérémie Chassaing, @thinkb4coding](https://twitter.com/thinkb4coding). As teased in that video, there will hopefully be a body of work that will describe the concept in more detail ... eventually (but hopefully not [inevitably](https://github.com/ylorph/The-Inevitable-Event-Centric-Book))...
+
+#### In Equinox
+
+The Equinox `type Decider`, which exposes an [API that covers the needs of making Consistent Decisions against a State derived from Events on a Stream](
+https://github.com/jet/equinox/blob/master/src/Equinox/Decider.fs#L22-L56)
+
+> NOTE the `Decider` itself in Equinox does no directly touch all three of the ingredients - while you pass it a `decide` function, the `initial` and `fold` functions, are  supplied to the specific store library (e.g. `Equinox.CosmosStore.CosmosStoreCategory`), as that manages the loading, snapshotting, syncing and caching of the state and events.
+
+#### In general
+
+While the concept of a Decider plays well with Event Sourcing and many different types of Store, its important to note that neither storage or event sourcing is a prerequisite. A lot of the value of the concept is that you can and should be able to talk about and implement one without reference to any specific store implementation (or even thinking about it ever being stored - it can also be used to manage in-memory structures such as UI trees etc).
+
+#### Consistency
+
+_In any system, any decision (or even query) processed by a Decider should be concurrency controlled_. If you're running in an Actor based system, the concurrency is managed by that. If you're building an in-memory decision system to support a game etc as Jérémie does in the talk, there's only one so that concern is side-stepped.
```

Comment:
```suggestion
_In any system, any decision (or even query) processed by a Decider should be concurrency controlled_. If you're running in an Actor based system, the concurrency is managed by that as one can spec if not more than one executor of decision logic can exist side-by-side. If you're building an in-memory decision system to support a game etc as Jérémie does in the talk, there's only one so that concern is side-stepped.
```